src/main/java/Team/TeamMember.java

```diff
@@ -0,0 +1,61 @@
+package Team;
+
+import equipments.Equipment;
+import equipments.EquipmentStatus;
+import equipments.technology.TechnologyEquipment;
+import equipments.uniform.UniformEquipment;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+public class TeamMember {
+    String name;
+    String teamName;
+    HashMap<String, List<UniformEquipment>> memberUniformEquipment;
+    HashMap<String, List<TechnologyEquipment>> memberTechnologyEquipment;
+
+    public TeamMember(String name, String teamName) {
+        this.name = name;
+        this.teamName = teamName;
+        memberUniformEquipment = new HashMap<String, List<UniformEquipment>>();
+        memberTechnologyEquipment = new HashMap<String, List<TechnologyEquipment>>();
```

Comment:
OCP- adding a third kind of equipment requires you to go through the whole code and changing many classes. also- what will you do with equipment that is both a piece of clothing and technology (such as a smartwatch)?